src/app/containers/StoryPromo/index.stories.jsx

```diff
@@ -71,4 +73,5 @@ storiesOf('Containers|Story Promo/AMP', module)
   .add('video fixture - amp', () => getAmpStoryPromo(videoFixture))
   .add('standard link promo - amp', () => getAmpStoryPromo(standardPromo))
   .add('video link promo - amp', () => getAmpStoryPromo(videoPromo))
-  .add('audio link promo - amp', () => getAmpStoryPromo(audioPromo));
+  .add('audio link promo - amp', () => getAmpStoryPromo(audioPromo))
+  .add('gallery link promo - amp', () => getCanonicalStoryPromo(galleryPromo));
```

Comment:
I think this should be getAmpStoryPromo(galleryPromo) instead of getCanonicalStoryPromo(galleryPromo)

Reply:
Yep, copy and paste.. 😄